src/pages/Filing/PrivacyNotice.tsx

```diff
@@ -10,7 +10,7 @@ function PrivacyNotice(): ReactElement {
       <Layout.Wrapper>
         <Layout.Content flushBottom>
           <CrumbTrail>
-            <Link href='/landing'>Platform home</Link>
+            <Link href='/landing'>Home</Link>
```

Comment:
Heyo! While you're updating this, there are a few bugs here that I think were recently introduced. ` href='/landing'` here won't work for users that aren't logged in. Try clicking the breadcrumb when logged out and it won't take you anywhere: should be `/`. If we add this page to the authenticated home page, then we can do something where it takes you to `landing` if you're logged in, but for now, it should be `/`.

Reply:
Breadcrumb updated to '/'